README.md

```diff
@@ -1,12 +1,35 @@
-# collaboration
+# Administration for TSC + Community Committee collaborative efforts
+This repository is formal efforts of the Node.js project that are overseen and require input from both the TSC and CommComm. 
 
-***ARCHIVE*: This repository is no longer active. Please see the the [Community Committee](https://github.com/nodejs/community-committee) repository for information on current efforts around supporting and growing the Node.js Community.**
+Initiatives that are overseen here:
+- [Travel Fund](https://github.com/nodejs/admin/blob/master/travel_fund.md)
 
-Facilitating Collaboration among the Node.js Working Groups.
+Repositories that fall under the responsibility of the TSC and CommComm:
 
-This team is just getting started. See [#1 - A Place To Start](https://github.com/nodejs/collaboration/issues/1) to follow the overall planning efforts.
+- this repo
+- (Getting Started)https://github.com/nodejs/getting-started
+- [Collaboration Summit](https://github.com/nodejs/summit)
+- (Moderation](https://github.com/nodejs/moderation)
 
-If you're curious about our proposed scope (or have some ideas on what it should be) see [#2](https://github.com/nodejs/collaboration/issues/2)
+## Governance and Current Members
 
-Want to help? [Request Team Access](https://github.com/orgs/nodejs/teams/collaboration) and visit 
-[#3](https://github.com/nodejs/collaboration/issues/3) to introduce yourself.
+The Admin repo is overseen by the [TSC](https://github.com/nodejs/TSC) and CommComm as collaborative groups. Any action requiring consensus and voting will abide by the following process.
+
+Members of the TSC and CommComm shall operate under Lazy Consensus as a collaborative unit in the Admin repo hereby referenced as the Admin members. Members shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
```

Comment:
Let's put "Admin members" in quotes to highlight it was a definition of the phrase.

Also, let's consider calling it "Admin Team Members" to disambiguate it from:
- the notion of it being related to GitHub "Owners" (which is sometimes referred to as "Admins")
- a Charted Working Group.

Reply:
-1 on quotes, but using Italics would work.. e.g. `*Admin members*`